ping-with-dsl/go/v0.23.0/Makefile

```diff
@@ -0,0 +1,5 @@
+image.json: Dockerfile main.go go.mod
```

Comment:
Long term, I see CI building these images, uploading them to some container registry, then committing this json file directly into the repo.

Reply:
I share the same vision apart from the committing to the repo part. Can you elaborate why do you think that is necessary?

Reply:
In general, I don't quite understand the purpose of the JSON files.

Reply:
> Can you elaborate why do you think that is necessary?

Depends on our use case, but maybe for this it's not worth it. It would be nice if you could pull the images built in CI if your code doesn't affect these things. It's like using it as a cache. 

> In general, I don't quite understand the purpose of the JSON files.

It's the simplest way to record the container image id. Since it's json we can import it in our typescript code directly as opposed to reading and parsing a file manually.